.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Deployment
+
+on:
+  release:
+    types: [published]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args:
+                      build -a -c --skip-tests 
```

Comment:
Shall we enable test cases at the deployment by adding github secrets?

Reply:
+1.